packages/taquito/src/context.ts

```diff
@@ -183,6 +187,10 @@ export class Context {
       // testnet and sandbox enviornment.   
       confirmationPollingInterval = confirmationPollingInterval.dividedBy(3);  
       this.config.confirmationPollingIntervalSecond = confirmationPollingInterval.toNumber();
+      
+      if (this.config.confirmationPollingIntervalSecond === 0) {
+        return 0.1;
+      }
```

Comment:
We should do the validation (that looks if the value is 0) before setting it in the config, otherwise, we will still use the 0 for the pollinginterval in the subsequent requests.
If `confirmationPollingInterval.toNumber() === 0`, we need to set the `confirmationPollingIntervalSecond` to 0.1 and not 0.

Reply:
Thanks, this is fixed.